### ISSUE TITLE
Some characters display incorrectly on the phone

### ISSUE THREAD

**Issue by bzaar** [morpher-ru/gramdict#140]
![image](https://user-images.githubusercontent.com/4028922/83640354-eba60600-a5a3-11ea-861d-0cdea676ae74.png)

**Comment by bzaar** [comment]
Для пик есть 2 символа: https://unicode-search.net/unicode-namesearch.pl?term=SPADE
Предлагаю взять светлый.

Взять ⌧ вместо ☒.

**Comment by bzaar** [comment]
Сюда же: мо⁺ жо⁺  - на телефоне ⁺ неправильно отображается.

Можно попробовать что-нибудь из [этого](https://unicode-search.net/unicode-namesearch.pl?term=PLUS%20SIGN) или ```<sup>+</sup>```.

**Comment by bzaar** [comment]
Оценка 4 часа.

**Comment by bzaar** [comment]
Вот еще вариант:

“⮽” U+2BBD Ballot Box with Light X Unicode Character

**Comment by GoToNightmare** [comment]
Нужно не только поменять фильтры, но и весь текст учебника, все подобные символы в тексте, верно?

**Comment by bzaar** [comment]
Да. И файл zalizniak.txt. Я потом изменю генератор этого файла, чтобы он генерировал выбранные тобой символы.
